modules/configuration/src/main/java/org/apache/ignite/configuration/internal/annotation/Validate.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.configuration.internal.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import org.apache.ignite.configuration.internal.validation.FieldValidator;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+/**
+ * This annotation applies custom validation to configuration field, for example:
+ * <pre name="code" class="java">
+ * public class ConfSchema {
+ *     {@literal @}Validate(SomeCustomValidator.class)
+ *     private String value;
+ * }
+ * </pre>
+ *
+ * If you need multiple custom validations:
+ * <pre name="code" class="java">
+ * public class ConfSchema {
+ *     {@literal @}Validate.List({
```

Comment:
No no no no no
You just write several annotations, that's it. That's the whole point of _repeatable_ annotations.

Reply:
Yeah, right, I actually use the correct way to repeat annotations in the example, just forgot about the doc